pykdtree/test_tree.py

```diff
@@ -385,33 +385,46 @@ def test_empty_fail():
     except ValueError as e:
         assert 'non-empty' in str(e), str(e)
 
-def test_tree_n_lt_maxint32_nk_gt_maxint32():
-    # n < UINT32_MAX but n * k > UINT32_MAX -> still uses 32-bit index
-    data_pts = np.random.random((2**20, 2)).astype(np.float32)
-    query_pts = np.random.random((3, 2)).astype(np.float32)
+@pytest.mark.skip(reason="Requires ~50G RAM")
+def test_tree_n_lt_maxint32_n_query_k_gt_maxint32():
+    # n_points < UINT32_MAX but n_query * k > UINT32_MAX -> still uses 32-bit index
```

Comment:
So this being skipped now, does that mean it wasn't properly testing things before?

Reply:
yes - previously it tested the case `n_points * k > UINT32_MAX`, but what it should have been testing was the case `n_query * k > UINT32_MAX`, since that is the case which previously overflowed